src/test/java/uk/gov/pay/directdebit/notifications/services/UserNotificationServiceTest.java

```diff
@@ -111,6 +115,31 @@ public void shouldSendMandateFailedEmailIfEmailNotifyIsEnabled() throws Exceptio
         );
     }
 
+    @Test
+    public void shouldSendMandateCancelledEmailIfEmailNotifyIsEnabled() throws Exception {
+        when(mockNotifyClientFactory.isEmailNotifyEnabled()).thenReturn(true);
+        userNotificationService = new UserNotificationService(mockDirectDebitConfig, mockNotifyClient, mockMetricRegistry, mockGatewayAccountService);
+
+        Mandate mandate = MandateFixture.aMandateFixture().toEntity();
+        HashMap<String, String> emailPersonalisation = new HashMap<>();
+        emailPersonalisation.put("mandate reference", mandate.getExternalId());
+        emailPersonalisation.put("org name", gatewayAccount.getServiceName());
+        emailPersonalisation.put("org phone", "+44 000-CAKE-000");
```

Comment:
It is not a valid number on purpose - it's a placeholder, because we need to get adminusers stuff in emails. Same for the address and so on. Is there a reason I should use a valid number (I can't think of any)?

Reply:
I was thinking about example phone numbers, but `+44 000-CAKE-000` - `+44 000-2253-000` will not resolve I think anyway :)